<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="Catch`1.xml" source-language="en-US" target-language="zh-CN">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac50f1b6c071e42c7a1058579fd3fca964ffad41b85.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">0f1b6c071e42c7a1058579fd3fca964ffad41b85</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.Statements.Catch`1">
          <source>The type of exception that the <ph id="ph1">&lt;see cref="T:System.Activities.Statements.Catch`1" /&gt;</ph> block is handling.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Activities.Statements.Catch`1" /&gt;</ph> 块正在处理的异常的类型。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Activities.Statements.Catch`1">
          <source>Contains the actions to be executed after an exception is raised in the corresponding Try block of a <ph id="ph1">&lt;see cref="T:System.Activities.Statements.TryCatchFinally" /&gt;</ph> activity.</source>
          <target state="translated">包含要在 <ph id="ph1">&lt;see cref="T:System.Activities.Statements.TryCatchFinally" /&gt;</ph> 活动的对应 Try 块中引发了异常之后执行的操作。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.Statements.Catch`1">
          <source>Once placed in the designer, <ph id="ph1">&lt;xref:System.Activities.Statements.Catch&gt;</ph> activities cannot be reordered in the designer.</source>
          <target state="translated">在设计器中，一次放置<ph id="ph1">&lt;xref:System.Activities.Statements.Catch&gt;</ph>活动不能在设计器中重新排序。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.Statements.Catch`1">
          <source>To reorder the collection of <ph id="ph1">&lt;xref:System.Activities.Statements.Catch&gt;</ph> activities, either delete and re-add them in the correct order, or use the xaml (code) view for the workflow.</source>
          <target state="translated">若要重新排序的集合<ph id="ph1">&lt;xref:System.Activities.Statements.Catch&gt;</ph>活动，请删除并重新将其添加按正确的顺序，或使用工作流 xaml （代码） 视图。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.Statements.Catch`1">
          <source>The following code sample demonstrates creating a <ph id="ph1">&lt;xref:System.Activities.Statements.Catch%601&gt;</ph> activity.</source>
          <target state="translated">下面的代码示例演示如何创建 <ph id="ph1">&lt;xref:System.Activities.Statements.Catch%601&gt;</ph> 活动。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.Statements.Catch`1">
          <source>This example is from the <bpt id="p1">[</bpt>Fault Handling in a Flowchart Activity Using TryCatch<ept id="p1">](~/docs/framework/windows-workflow-foundation/samples/fault-handling-in-a-flowchart-activity-using-trycatch.md)</ept> sample.</source>
          <target state="translated">此示例摘自<bpt id="p1">[</bpt>故障处理中流程图活动使用 TryCatch<ept id="p1">](~/docs/framework/windows-workflow-foundation/samples/fault-handling-in-a-flowchart-activity-using-trycatch.md)</ept>示例。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Activities.Statements.Catch`1.#ctor">
          <source>Creates a new instance of the <ph id="ph1">&lt;see cref="T:System.Activities.Statements.Catch`1" /&gt;</ph> class.</source>
          <target state="translated">创建 <ph id="ph1">&lt;see cref="T:System.Activities.Statements.Catch`1" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Activities.Statements.Catch`1.#ctor">
          <source>The following code sample demonstrates creating a <ph id="ph1">&lt;xref:System.Activities.Statements.Catch%601&gt;</ph> activity.</source>
          <target state="translated">下面的代码示例演示如何创建 <ph id="ph1">&lt;xref:System.Activities.Statements.Catch%601&gt;</ph> 活动。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Activities.Statements.Catch`1.#ctor">
          <source>This example is from the <bpt id="p1">[</bpt>Fault Handling in a Flowchart Activity Using TryCatch<ept id="p1">](~/docs/framework/windows-workflow-foundation/samples/fault-handling-in-a-flowchart-activity-using-trycatch.md)</ept> sample.</source>
          <target state="translated">此示例摘自<bpt id="p1">[</bpt>故障处理中流程图活动使用 TryCatch<ept id="p1">](~/docs/framework/windows-workflow-foundation/samples/fault-handling-in-a-flowchart-activity-using-trycatch.md)</ept>示例。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="P:System.Activities.Statements.Catch`1.Action">
          <source>The handler for the exception being caught.</source>
          <target state="translated">所捕获异常的处理程序。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.Statements.Catch`1.Action">
          <source>The activity action.</source>
          <target state="translated">活动操作。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.Statements.Catch`1.Action">
          <source>The exception handler is represented by a <ph id="ph1">&lt;xref:System.Activities.ActivityAction%601&gt;</ph> object.</source>
          <target state="translated">异常处理程序由 <ph id="ph1">&lt;xref:System.Activities.ActivityAction%601&gt;</ph> 对象表示。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.Statements.Catch`1.Action">
          <source>The following code sample demonstrates assigning the Action property of a <ph id="ph1">&lt;xref:System.Activities.Statements.Catch%601&gt;</ph> activity.</source>
          <target state="translated">下面的代码示例演示如何分配 <ph id="ph1">&lt;xref:System.Activities.Statements.Catch%601&gt;</ph> 活动的 Action 属性。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.Statements.Catch`1.Action">
          <source>This example is from the <bpt id="p1">[</bpt>Fault Handling in a Flowchart Activity Using TryCatch<ept id="p1">](~/docs/framework/windows-workflow-foundation/samples/fault-handling-in-a-flowchart-activity-using-trycatch.md)</ept> sample.</source>
          <target state="translated">此示例摘自<bpt id="p1">[</bpt>故障处理中流程图活动使用 TryCatch<ept id="p1">](~/docs/framework/windows-workflow-foundation/samples/fault-handling-in-a-flowchart-activity-using-trycatch.md)</ept>示例。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="P:System.Activities.Statements.Catch`1.ExceptionType">
          <source>The type of exception that the <ph id="ph1">&lt;see cref="T:System.Activities.Statements.Catch" /&gt;</ph> block is handling.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Activities.Statements.Catch" /&gt;</ph> 块正在处理的异常的类型。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.Statements.Catch`1.ExceptionType">
          <source>The exception type.</source>
          <target state="translated">异常类型。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>